microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,318 @@
+   ![In Solution Explorer for the WebView2 UWP project, selecting 'Add Reference'.](winrt-from-js-images/sln-xpl-wv2-uwp-add-ref.png)
+
+1. In the tree on the left, select the **Projects** node, and then select the **WinRTAdapter** checkbox:
+
+   ![Alt-text 10.](winrt-from-js-images/10.png)
+
+1. Click the **OK** button to close the dialog.
+
+1. Right-click the **WinRTAdapter** project, and then select **Build**.
+
+Source code is generated for the selected API items.
+
+
+<!-- =============================================== -->
+## Step 7. Add the host object in the main project
+
+1. In Solution Explorer, expand the **webview2_sample_uwp** project, and then select **MainPage.xaml.cs**.
+
+1. Below the `MainPage` constructor, add the following method, to pass the WinRT object to the host app's web side:
+
+   ```csharp
+   private async void InitializeWebView2Async()
+   {
+      await WebView2.EnsureCoreWebView2Async();
+      var dispatchAdapter = new WinRTAdapter.DispatchAdapter();
+      WebView2.CoreWebView2.AddHostObjectToScript("Windows", dispatchAdapter.WrapNamedObject("Windows", dispatchAdapter));
+   }
+   ```
+    
+1. In the `MainPage` constructor, above the `StatusUpdate("Ready");` line, add the following code:
+
+   ```csharp
+   InitializeWebView2Async();
+   ```
+
+1. Right-click the webview2_sample_uwp project and then select **Set as startup project**.
+
+1. Press `Shift+Ctrl+S` to save all files.
+
+1. Press `F5` to run the sample app.
+
+The code now has your host object on the web side.
+
+
+<!-- =============================================== -->
+## Step 8. Call APIs of the host object from the web side
+
+Next, use DevTools to demonstrate that web-side code can potentially call the included, specified host-side API items.
+
+1. Click in the main part of the WebView2 sample app window to give it focus, and then press `Ctrl+Shift+I` to open Microsoft Edge DevTools.  Or, right-click the page, and then select **Inspect**.  The Microsoft Edge DevTools window opens.
+
+1. If the Microsoft Edge DevTools window isn't visible, press `Alt+Tab` to display the DevTools window.
+
+1. Under the **Console** pane of DevTools, paste the following code to confirm that your host app's web-side code can call the app's host object's methods.  To do this, call `Language.displayName`, as follows:
+
+   ```javascript
+   (new chrome.webview.hostObjects.sync.Windows.Globalization.Language("en-US")).displayName
+   ```
+
+   The **Console** outputs `English (United States)`, or another language, demonstrating that your app's host-side code can be called from web-side code.
+
+   ![Using the DevTools Console to test calling native-side code from web-side code.](winrt-from-js-images/devtools-console-calling-native-side-code.png)
+
+1. Close the UWP sample app.  The DevTools window also closes.
+
+
+<!-- ====================================================================== -->
+## WebView2 Properties available in the WinRTAdapter Property Pages
+
```

Comment:
@jasonstephen15 - Doc these property rows from command-line help strings?  re: WebView2 Properties available in the WinRTAdapter Property Pages